WixInstaller/XenCenter.wxs

```diff
@@ -35,7 +35,7 @@
      xmlns:netfx="http://schemas.microsoft.com/wix/NetFxExtension">
     <?include branding.wxi ?>
     <Product Id="$(var.ProductCode)" Name="$(var.BrandConsole)" Language="$(env.WixLangId)" Version="$(var.ProductVersion)" Manufacturer="$(var.CompanyNameLegal)" UpgradeCode="$(var.UpgradeCode)">
-        <Package Languages="1033,1041,2052,1028" Description="$(var.BrandConsole)" Comments="none." InstallerVersion="200" Compressed="yes" />
+        <Package Languages="1033" Description="$(var.BrandConsole)" Comments="none." InstallerVersion="200" Compressed="yes" />
```

Comment:
Im assuming here that languages=number is a reference to a specific language dropped due to the localisation depreciation?

Reply:
that's right. I removed support everywhere but here!

List of codes can be seen [here](https://github.com/xenserver/xenadmin/blob/91b224d0c42a776b4d1c49c3cec84688879928e9/WixInstaller/codepagechange.vbs)